randovania/games/super_metroid/json_data/Maridia.txt

```diff
@@ -698,12 +698,12 @@ Extra - asset_id: 147
 > Door from Aqueduct; Heals? False; Default Node
   * Layers: default
   * Permanently Locked to Aqueduct/Door to West Aqueduct Quicksand Room
-  > Door to West Sandtrap
+  > Door to West Sand Hole
       Trivial
 
```

Comment:
I didn't think this one would get changed. I've never heard it called any other name.